IBAnimatable/PresenterManager.swift

```diff
@@ -19,12 +19,12 @@ class PresenterManager {
   }
   
   // MARK: - Private
-  private var cache = [TransitionAnimationType: Presenter]()
+  private var cache = [String: Presenter]()
```

Comment:
@tbaranes Sorry, I may confuse you, I think using `String` as the key is good, and we save `TransitionAnimationType.stringValue` as the value of the key, it is unique. I like your solution here.